### DISCUSSION TITLE
Block access to changes my repo

### DISCUSSION THREAD

**Discussion by AvivLevinson** [community/community#22912] [New to GitHub]
<p>How to prevent changes in my repository without  user permission</p>

**Comment by allenkinzalow** [comment, marked as answer]
<p><a href="https://help.github.com/articles/access-permissions-on-github/" rel="nofollow noopener">https://help.github.com/articles/access-permissions-on-github/</a></p>
<p>If it is a user repository there appears to only be Owners &amp; Collaborators and there appears to be a defined list of things a collaborator can do: <a href="https://help.github.com/articles/permission-levels-for-a-user-account-repository/" rel="nofollow noopener">https://help.github.com/articles/permission-levels-for-a-user-account-repository/</a></p>
<p>It also doesnt appear like you can change these permissions.</p>
